### ISSUE TITLE
Question about lists

### ISSUE THREAD

**Issue by emaschi5** [dtankdempse/daddylive-m3u#25]
Thanks for the great job, i see that now the playlist contain also the daily events, so now the links are not static anymore right? 
i've only italian channels but i want also the live events

**Comment by dtankdempse** [comment]
Yes, only the events change daily, the rest of the list remains the same.

I'm not sure what you mean by: 'I've only Italian channels, but I would love to have the live events as well.'?

All events provided by the Daddylive Schedule API should be included in the list.

**Comment by emaschi5** [comment]
I’ve tivimate playlist with only Italian channels, but I can’t use this mod playlist because now the event change daily. No problem I will use the all channel and save the Italian on favourite

**Comment by dtankdempse** [comment]
Oh, I see. In TiviMate, you can easily manage groups by disabling the ones you don't want. Here's a video showing how it's done: https://www.youtube.com/watch?v=C4KwOKfRvwg&ab_channel=sfkjeld

**Comment by emaschi5** [comment]
Unfortunately not in the free version
